inline void Parser::exceptionHandler(std::exception const &exc)         
{
    throw;
}
